src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64XFormsDotNet.apkdesc

```diff
@@ -2486,5 +2516,5 @@
       "Size": 812848
     }
   },
-  "PackageSize": 10521867
+  "PackageSize": 11874565
```

Comment:
Investigating what the size increase here is, and if it is actually correct.

Reply:
One of them is:
* https://github.com/xamarin/Xamarin.Forms/blob/2f8f4864a4d289dc89a6228e2ca9d6a49993e365/Xamarin.Forms.Platform.Android/PowerSaveModeBroadcastReceiver.cs#L6-L7

```csharp
[BroadcastReceiver(Enabled = true, Exported = false)]
public class PowerSaveModeBroadcastReceiver : BroadcastReceiver
```

It _is_ managed code, I suppose it should not be linked away?

Reply:
Actually testing here to get a better diff:
* https://github.com/dotnet/android/pull/9101

I don't know what the current state is of main, it might already have `PowerSaveModeBroadcastReceiver`.